prompt_toolkit/widgets/base.py

```diff
@@ -373,10 +373,17 @@ class Button:
     """
 
     def __init__(
-        self, text: str, handler: Optional[Callable[[], None]] = None, width: int = 12
+        self,
+        text: str,
+        handler: Optional[Callable[[], None]] = None,
+        width: int = 12,
+        left_char: str = "<",
```

Comment:
One more change request on this. Would it be possible to rename ``left_char`` and ``right_char`` to ``left_symbol`` and ``right_symbol``? Just for consistency with ``layout/margins/ScrollbarMargin``.

Reply:
Sure, I'll do it later

Reply:
Thanks!

Reply:
Done!